libcontainer/rootfs_linux.go

```diff
@@ -107,12 +121,12 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 		if err := os.MkdirAll(dest, 0755); err != nil {
 			return err
 		}
-		return syscall.Mount(m.Source, dest, m.Device, uintptr(m.Flags), "")
+		return mountPropagate(m.Source, dest, m.Device, m.Flags, "", m.PropagationFlags)
```

Comment:
As we seem to be modifying both m.Dest and m.Data (if need be), I would rather leave it as it is and not try to define a method on configs.Mount(). 

Anyway, that can be done later as part of cleanup and should not be a blocker for this patch.